src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -1409,6 +1420,97 @@ def disable_features(cmd, client, resource_group_name, cluster_name, features, k
     return str.format(consts.Successfully_Disabled_Features, features, connected_cluster.name)
 
 
+def troubleshoot(cmd, client, resource_group_name, cluster_name, kube_config=None, kube_context=None, location=None, storage_account=None,
```

Comment:
Is it okay to mandate the cluster_name and rg_name. A customer can run into onboarding issue even before the PUT cc call. For them, these paramters won't make any sense

Reply:
Are we saying that customer's should run the troubleshoot cmd only when they see failures in 'helm install' step?

Reply:
If they face any issue onboarding, they will run this command, with the same -n and -g, In the troubleshoot, we are anyway doing the get on that, which will be null if onboarding to ARM itself didnt happen.

The troubleshoot has to be run when they face any issue.

Reply:
should be fine i guess